src/services/cgnOperatorSearchService.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * This service interactsnwith the Bonus API
+ */
+
+import {
+  IResponseErrorInternal,
+  IResponseErrorNotFound,
+  IResponseErrorValidation,
+  IResponseSuccessJson,
+  ResponseErrorInternal,
+  ResponseErrorNotFound,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { Merchant } from "../../generated/cgn-operator-search/Merchant";
+import {
+  ResponseErrorStatusNotDefinedInSpec,
+  withCatchAsInternalError,
+  withValidatedOrInternalError
+} from "../utils/responses";
+import { readableProblem } from "../utils/errorsFormatter";
+import { CgnOperatorSearchAPIClient } from "../../src/clients/cgn-operator-search";
+import { OnlineMerchantSearchRequest } from "../../generated/io-cgn-operator-search-api/OnlineMerchantSearchRequest";
+import { OnlineMerchants } from "../../generated/cgn-operator-search/OnlineMerchants";
+import { OfflineMerchantSearchRequest } from "../../generated/io-cgn-operator-search-api/OfflineMerchantSearchRequest";
+import { OfflineMerchants } from "../../generated/cgn-operator-search/OfflineMerchants";
+export default class CgnService {
+  constructor(
+    private readonly cgnOperatorSearchApiClient: ReturnType<
+      CgnOperatorSearchAPIClient
+    >
+  ) {}
+
+  /**
+   * Get the CGN operator/merchant identified by its identifier.
```

Comment:
```suggestion
   * Get the CGN operator/merchant by its identifier.
```

Reply:
done in https://github.com/pagopa/io-backend/pull/804/commits/7d342ba24b1efeb396af1168d077070de40b0b3e